CONTRIBUTING.md

```diff
@@ -36,12 +36,87 @@ library, consider developing it in a separate git repository. When it is ready t
 integrated, the Solana maintainers will work with you to decide on a path forward. Smaller
 libraries may be copied in whereas very large ones may be pulled in with a package manager.
 
+Getting Pull Requests Merged
+---
+
+There is no single person assigned to watching GitHub PR queue and ushering you through
+the process. Typically, you will ask the person that wrote a component to review
+wrote a component will review changes to it. You can find the author using `git blame`
```

Comment:
"the person that wrote a component to review wrote a component will review changes to it"... needs attn

Reply:
Fixed in c4ea7c0